src/pages/post-test-summary/post-test-summary.ts

```diff
@@ -53,4 +55,30 @@ export class PostTestSummaryPage {
     });
     modal.present();
   }
+
+  showRouteListAlert() {
+    const inputs = [];
+    for (let i = 0; i < 13; i += 1) {
+      inputs.push({
+        type: 'radio',
+        label: i,
```

Comment:
I think you need to 1-index this (I've not seen route 0 used)

Reply:
yep, good point.
Done